docs/tutorial/kymotracking.rst

```diff
@@ -398,7 +398,31 @@ This returns a tuple of lags and MSD estimates. If we only wish MSDs up to a cer
     >>> tracks[0].msd(max_lag = 5)
     (array([0.16, 0.32, 0.48, 0.64, 0.8 ]), array([ 3.63439512,  6.13181603,  9.08823918, 11.43574189, 12.61152129]))
 
-MSDs are typically used to calculate diffusion constants.
+Similarly, we can compute an averaged MSD estimate for a number of tracks in a :class:`~lumicks.pylake.kymotracker.kymotrack.KymoTrackGroup` using :func:`~lumicks.pylake.kymotracker.kymotrack.KymoTrackGroup.ensemble_msd`::
+
+    ensemble_msd = tracks.ensemble_msd()
+
+This returns a weighted average of the MSDs coming from all the tracks (where the weight is determined by the number of points that contribute to each lag).
+Note that if the tracks are of equal length, this weighting will not have an effect (all the weights will be the same).
+The implicit assumption here is that all tracks are diffusive and their diffusion constants are the same.
+You can plot the ensemble msd as follows::
+
+    ensemble_msd.plot()
+
+The following properties can be read from :class:`~lumicks.pylake.kymotracker.detail.msd_estimation.EnsembleMSD`:
```

Comment:
I'm wondering if we should expose objects from within detail in the docs? Unfortunately, I actually do not have a good idea how to circumvent this and still provide the end user with a good documentation.

Reply:
It's possible to move that class to a public space, but I don't like that it would clutter up the API with something users never construct. For now this seems like the least of all evils to me.